src/CarbonAware.CLI/src/CarbonAware.CLI.csproj

```diff
@@ -24,30 +22,40 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="CommandLineParser" Version="2.8.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="6.0.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="System.ComponentModel.Composition" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\CarbonAware\src\CarbonAware.csproj" />
     <ProjectReference Include="..\..\CarbonAware.Aggregators\src\CarbonAware.Aggregators.csproj" />
-</ItemGroup>
+    <ProjectReference Include="..\..\CarbonAware\src\CarbonAware.csproj" />
+  </ItemGroup>
 
- <ItemGroup>
+  <ItemGroup>
+    <Compile Update="CommandOptions.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>CommandOptions.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="CommandOptions.resx">
```

Comment:
Consider adding a namepace like [here](https://github.com/dotnet/sdk/blob/a43e07be4bd1d38824adf56e201d845c616716a2/src/Cli/dotnet/dotnet.csproj#L35) so you can more easily reference in code like [this](https://github.com/dotnet/sdk/blob/main/src/Cli/dotnet/commands/dotnet-build/BuildCommandParser.cs#L24)